CHANGELOG.md

```diff
@@ -11,56 +13,42 @@ All notable changes to this project will be documented in this file.
 - Make `CertificateReservedNodeCommonName` dynamically reloadable [EventStore#3966](https://github.com/EventStore/EventStore/pull/3966)
 - Log levels [EventStore#3973](https://github.com/EventStore/EventStore/pull/3973)
 - Updated reference to EventStore.Plugins to 23.10.1 [EventStore#3990](https://github.com/EventStore/EventStore/pull/3990)
-
 - No longer log misleading error about 'Max internal streams limit' when running out of file handles [EventStore#3902](https://github.com/EventStore/EventStore/pull/3902)
 
 ### Removed
 
 - Unused code around checkpoints [EventStore#3900](https://github.com/EventStore/EventStore/pull/3900)
 - Removed unused code paths [EventStore#3928](https://github.com/EventStore/EventStore/pull/3928)
+- PersistentConfig1 system event type name to $PersistentConfig1 [EventStore#3932](https://github.com/EventStore/EventStore/pull/3932)
```

Comment:
Renamed ?

Reply:
Ah yeah, that shouldn't be in the "Removed" section. I've updated the entry and moved it to "Changed"

Reply:
ah, we removed the 1 from the end as well